### PR TITLE
Metrics for GraphQL endpoints

### PR DESCRIPTION
**Pull request by ivantenevvasilev**
**Description**

Changes proposed in this pull request:
- introduce graphql endpoint usage metrics
- introduce graphql mutation and query usage charts
- update Compass/Director dashboard

**Pull Request status**
- [x] Implementation
- [x] `chart/compass/values.yaml` is updated

### CODE REVIEW

File: components/director/internal/metrics/collector.go [hunk 1]
```diff
@@ -24,6 +25,7 @@ type Collector struct {
 	hydraRequestTotal      *prometheus.CounterVec
 	hydraRequestDuration   *prometheus.HistogramVec
 	clientTotal            *prometheus.CounterVec
+	mutationCount          *prometheus.CounterVec
```
Comment (StanislavStefanov) [inline components/director/internal/metrics/collector.go:28]: We are using this counter for both graphql mutations and queries so `gqlOperationCount` may be mora appropriate name

**Review by StanislavStefanov — CHANGES_REQUESTED** [review]

**Review by StanislavStefanov — APPROVED** [review]

File: chart/compass/values.yaml [hunk 2]
```diff
@@ -182,6 +182,7 @@ global:
     metrics:
       port: 3003
       enableClientInstrumentation: true
+      enableGraphqlOperationInstrumentation: false
```
Comment (PetarTodorovv) [inline chart/compass/values.yaml:185]: why this property is false? Shouldn't we use it in production environment as well?

Comment (ivantenevvasilev) [reply chart/compass/values.yaml:185]: I was under the impression, that it's better to have it off by default and have it explicitly overridden to be enabled wherever it is needed.

Comment (PetarTodorovv) [reply chart/compass/values.yaml:185]: Yep, generally that's the idea but If we're gonna use this functionality in production(everywhere) then I think we can switch it to `true` by default. That way we'll have it configured only at one place, less overrides and cleaner setup.

Usually we use the "default false" approach when we want the productive configuration to be `false` and only override it to `true` for local setup

**Review by StanislavStefanov — APPROVED** [review]

**Review by StanislavStefanov — APPROVED** [review]

**Review by StanislavStefanov — APPROVED** [review]

**Pull request merged**